lib/cocoapods/generator/xcconfig/aggregate_xcconfig.rb

```diff
@@ -68,12 +68,26 @@ def generate
           # that use Swift. Setting this for the embedded target would
           # cause an App Store rejection because frameworks cannot be embedded
           # in embedded targets.
-          if !target.requires_host_target? && pod_targets.any?(&:uses_swift?)
-            config['EMBEDDED_CONTENT_CONTAINS_SWIFT'] = 'YES'
-            config['ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES'] = 'YES'
+
+	  # Extract swift_version number
+	  @swift_version = target.build_configurations.build_settings['SWIFT_VERSION']
+          @swift_version_check = 3.0
```

Comment:
So to recap: if you see `EMBEDDED_CONTENT_CONTAINS_SWIFT`, use that setting, otherwise, use the newer `ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES` setting.